dipy/segment/tissue.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+from dipy.sims.voxel import add_noise
+from dipy.segment.mrf import (ConstantObservationModel,
+                              IteratedConditionalModes)
+
+
+class TissueClassifierHMRF(object):
+    """"
+    This class contains the methods for tissue classification using the Markov
+    Random Fields modeling approach
+    """
+
+    def __init__(self, save_history=False, verbose=True):
+        
+        self.save_history = save_history
+        self.segmentations = []
+        self.pves = []
+        self.energies = []
+        self.energies_sum = []
+        self.verbose = verbose
+
+        pass
+
+    def classify(self, image, nclasses, beta, max_iter):
+        """
+        This method uses the Maximum a posteriori - Markov Random Field
+        approach for segmentation by using the Iterative Conditional Modes and
+        Expectation Maximization to estimate the parameters.
+        
+        Parameters
+        ----------
+        image : 3D ndarray, T1 structural image.
+        nclasses : int, number of desired classes.
+        beta : float, smoothing parameter, the higher this number the 
+                smoother the output will be.
+        max_iter : number of desired iterations. Ususally between 0 and 0.5
```

Comment:
``max_iter`` docstring is wrong. Please correct. Say int first and then explain below.